plugins/rtmp-services/data/package.json

```diff
@@ -1,11 +1,11 @@
 {
-	"$schema": "schema/package-schema.json",
-	"url": "https://obsproject.com/obs2_update/rtmp-services",
-	"version": 197,
-	"files": [
-		{
-			"name": "services.json",
-			"version": 197
-		}
-	]
-}
+    "$schema": "schema/package-schema.json",
+    "url": "https://obsproject.com/obs2_update/rtmp-services",
+    "version": 197,
+    "files": [
+        {
+            "name": "services.json",
+            "version": 197
+        }
+    ]
+}
```

Comment:
End files with a newline. If this was automated, we may need to adjust the formatter.

Reply:
Why? JSON data is generally not terminated with a newline.

Reply:
IIRC, _some_ Linux tools expect all files to end with a newline character, otherwise the last line is not treated like a full line and is never read/parsed correctly. I don't know if this is a historical artifact that is no longer relevant, or if there are still some tools for which this is true.

Also, GitHub's web UI yells at you about it.

Reply:
I believe it's true of a variety of tools like `wc` (word count), `diff` will work but yell at you. I suppose one valid argument is that - sadly - we're hand editing these data files, so while technically JSON is terminated with `}` and a newline is superfluous some editors will automatically add a newline. So I guess for compatibility with those we'll have to have one (if we want cleaner diffs).